test/e2e/podtimingcontroller/podtimingcontroller.go

```diff
@@ -61,20 +61,23 @@ func (r *podTimingReconciler) reconcile(ctx context.Context, req ctrl.Request) e
 		return err
 	}
 
-	phase := pod.Status.Phase
-	podPhaseIdentifier := string(phase) + pod.Namespace + pod.Name + string(pod.UID)
+	if !isPodReady(&pod) {
+		return nil
+	}
+
+	podIdentifier := pod.Namespace + pod.Name + string(pod.UID)
 	alreadyRecorded := func() bool {
 		r.alreadyRecordedLock.RLock()
 		defer r.alreadyRecordedLock.RUnlock()
 
-		return r.alreadyRecorded.Has(podPhaseIdentifier)
+		return r.alreadyRecorded.Has(podIdentifier)
 	}()
 	if alreadyRecorded {
 		return nil
 	}
 
 	enterPhaseDuration := time.Since(pod.CreationTimestamp.Time).Seconds()
```

Comment:
can we rename all phases references e.g enterPhaseDuration, podEnterPhaseDurationSeconds...?

Reply:
@enxebre good point, updated, ptal